src/reducers/requests/plugins.js

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+// @flow
+
+import {combineReducers} from 'redux';
+import {PluginTypes} from 'action_types';
+
+import {handleRequest, initialRequestState} from './helpers';
+
+import type {GenericAction} from '../../types/actions';
+import type {RequestStatusType, PluginsRequestsStatuses} from '../../types/requests';
+
+function installMarketplacePlugin(state: RequestStatusType = initialRequestState(), action: GenericAction): RequestStatusType {
```

Comment:
Do we need a reducer and store this status in the store at all? The component is just making the call [directly](https://github.com/mattermost/mattermost-webapp/blob/23648a1d3cb8c3f31adcc70a37056111afcc9f24/components/plugin_marketplace/marketplace_item/marketplace_item.jsx#L59), receiving the error back directly and it never references anything in the store.

Reply:
it's a tiny bit nicer to have the reducer as it has some built in functionality to logout https://github.com/mattermost/mattermost-redux/blob/ef51b0056f5aa6613827e1475cde1339320aeef2/src/actions/helpers.js#L88. Also allows requests to be cancelled if necessary which is a similar problem we ran into when searching through the get API

Reply:
In that case you can still keep the action call which is the one that uses `bindClientFunc` where you have that built in functionality to logout. 

However, I still do not see the value of having this reducer that `handleRequest` and manages the state for the status returned from the client call.